explorer/src/components/account/NFTokenAccountHeader.tsx

```diff
@@ -0,0 +1,113 @@
+import { NftokenTypes } from "@glow-xyz/nftoken-js";
+import "bootstrap/dist/js/bootstrap.min.js";
+import React from "react";
+import { Account } from "../../providers/accounts";
+import {
+  parseNFTokenCollectionAccount,
+  parseNFTokenNFTAccount,
+} from "../../providers/accounts/utils/isNFTokenAccount";
+import { InfoTooltip } from "../common/InfoTooltip";
+import { CachedImageContent } from "../common/NFTArt";
+import { useNftokenMetadata } from "./nftoken-hooks";
+
+export function NFTokenAccountHeader({ account }: { account: Account }) {
+  const nft = parseNFTokenNFTAccount(account);
```

Comment:
There's a lot of render-time parsing going on in this PR. Since these operations are expensive (lots of ferrying data into and out of buffers, and base58 encoding things over and over) consider `useMemo()` so that you only have to calculate this stuff _once_ then you render with the result over and over.

Reply:
Hmm, it's really not a performance hit tho... Have you noticed any issues loading the page? The parsing is very fast.